xla/service/gpu/transforms/layout_assignment.cc

```diff
@@ -493,6 +624,11 @@ absl::Status GpuLayoutAssignment::AddBackendConstraints(
           Cast<HloCustomCallInstruction>(instruction), constraints));
     }
 
+    if (IsCuDNNConvolutionFusion(*instruction)) {
```

Comment:
There's already IsGpuFusionKind() in `service/gpu/hlo_fusion_analysis.h`, could it be reused?